index.html

```diff
@@ -101,7 +96,252 @@ <h1>I'm Adeyemi Glad to see you!</h1>
                 </li>  
               </ul>            
             </div>
-            <div> 
+          </div>
+        </section>
+        <Section class="Works">
+          <div class="cardWork">
+            <img id="snap_portfolio">
+            <div class="leftBlock">
+              <div class="pryText_head">
+                <div class="text_head">
+                  Tonic
+                </div>
+              <div class="frame_card">
+                <div class="Client_card">
+                  <div class="Client_txt">
+                    CANOPY
+                  </div>
+                </div>
+                <span class="Counter"></span>
+                <div class="Role_card">
+                  <div class="Role_txt">
+                    Back End Dev
+                  </div>
+                </div>
+                <span class="Counter2"></span>
+                <div class="Year_card">
+                  <p class="Year_txt">
+                    2015
+                  </p>
+                </div>
+              </div>
+            </div>
+            <div class="pryText_body">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+              </p>
+            </div>
+            <ul class="Tags">
+              <div class="textTag_Hml">
+                <li >
+                  <div class="Hml">
+                    <p>html</p>
+                    </div>
+                </li>
+              </div>
+              <div class="textTagcss">
+                <li>
+                  <div class="Css">
+                    <p>css</p>
+                  </div>
+                </li>
+              </div>
+              <div class="textTagjava">
+                <li>
+                  <div class="java">
+                    <p>javascript</p>
+                    </div>
+                </li>
+              </div>
+            </ul>
+              <div id="Action_bttn">
+                <div class="projbttn">
+                  <div class="click_bttn">
+                  <a href="#SeeProject" >See Project</a>
```

Comment:
Kindly style your button link in line with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) template.

| Your Design |Figma Design |
| :---:   | :---: | 
|![btn](https://user-images.githubusercontent.com/73854999/200114280-0eb87c2f-6f69-4c59-aefb-527d2af34a71.PNG)| ![mic-btn-style](https://user-images.githubusercontent.com/73854999/200114270-39255bd8-c016-4da2-bf98-178f5f5a7e56.PNG)|